packages/server/lib/server-e2e.ts

```diff
@@ -190,7 +190,9 @@ export class ServerE2E extends ServerBase<SocketE2E> {
       }
 
       // @ts-ignore
-      return !!getRouteForRequest(this.netStubbingState?.routes, proxiedReq)
+      const iterator = getRoutesForRequest(this.netStubbingState?.routes, proxiedReq)
+
+      return !iterator.next().done
```

Comment:
Generators aren't used all that often in this code-base; a comment here explaining what this logic is validating could be helpful.

Reply:
Yup, I find myself consulting MDN everytime I use the iterator API. I've added a comment and a (hopefully) reasonably named variable to make this less obscure.